.travis.yml

```diff
@@ -28,7 +28,7 @@ before_script:
 jobs:
   include:
     - stage: Build
-    - script:
+      script:
```

Comment:
```
     - stage: Test
     - script:
```
->
```
     - stage: Test
       script:
```

Reply:
yeah tried it that way once and didn't get the right result..

Reply:
probably because I had not yet removed the -go.. looks good now